### PR TITLE
Added BeezLabs to the Adopters list

### PR DESCRIPTION
**Pull request by karthikeyan23**
Signed-off-by: Karthik <kn@beezlabs.com>

## Hey, I just made a Pull Request!

Added BeezLabs to the Adopters List

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -107,3 +107,4 @@ _If you're using Backstage in your organization, please try to add your company
...
 | [Bonial International GmbH](https://www.bonial.com/)                        | [@pjungermann](https://github.com/pjungermann)                                                                                                                                                                                                                   | Centralized developer portal with software catalog, tech docs, templates, and more.                                                                                                                                                                                                                                                                  |
+| [Beez Innovation Labs Pvt. Ltd](https://www.beezlabs.com.com/) | [Karthikeyan Venkatesan](https://github.com/karthikeyan23) | Developer portal with software catalog, scaffolding, tech docs, templates, and infra. |
```
Comment (jhaals) [inline ADOPTERS.md:110]: It looks like there snuck in an additional `.com` in the url :)

**Review by jhaals — COMMENTED** [review]
Welcome again! :tada: This one passes DCO but there's a tiny mistake in the link to your website

File: ADOPTERS.md (hunk 1 above)
Comment (Rugvip) [reply ADOPTERS.md]: Fix'd x)

**Review by Rugvip — APPROVED** [review]

File: ADOPTERS.md (hunk 1 above)
Comment (jhaals) [reply ADOPTERS.md]: Oh yes!

**Comment by jhaals** [comment]
🚢

File: ADOPTERS.md (hunk 1 above)
Comment (karthikeyan23) [reply ADOPTERS.md]: Thanks a lot @jhaals @Rugvip 😉

**Pull request merged**